client.go

```diff
@@ -59,6 +59,22 @@ func (c *Client) Store(agmi AgencyMonthlyInfo) error {
 	return nil
 }
 
+// Store stores an agreggation in the database.
+func (c *Client) StoreAgreggation(agreggation Agreggation) error {
```

Comment:
Renomear parâmetro para `package`

```suggestion
func (c *Client) StorePackage(agreggation Agreggation) error {
```

Reply:
O parâmetro não pode ser renomeado para package pois é uma palavra reservada do golang